.env.template

```diff
@@ -5,4 +5,10 @@ FLASK_ENV=development
 #Fill in the bellow for your trello access
 DEVELOPER_API_KEY="<your key>"
 MY_SECRET="<your secret>"
-SERVER_TOKEN="<your token>"
\ No newline at end of file
+SERVER_TOKEN="<your token>"
+
+MONGO_USER="dbUser"
+MONGO_USER_PW="wZJQpucTJ68MXUFF"
+MONGO_CONNECTION="mongodb+srv"    
+MONGO_SERVER="cluster0.snxma.mongodb.net"
+MONGO_DATABASE="myFirstDatabase"
```

Comment:
I wasn't able to connect, but these look like legitimate details for your database - if that's correct we should remove them and rotate the databases password as these are now public! (And if they're not, I'd suggest switching the password to an obviously false one for future to prevent the same question arising!)